### PR TITLE
CORE-1016: added a new template for unrecoverable errors in the event recorder.

### PR DESCRIPTION
**Pull request by slr71**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,6 +8,7 @@ bin
 .eastwood
 .project
 .lein*
+.nrepl-port
```
Comment (slr71) [inline .gitignore:11]: Emacs creates this file when you use the CIDER REPL.

**Comment by sri2k1us** [comment]
Couples of things to look into:

- Now is a good time to remove / replace `iplant`
- Allow rich text formatting in message body

**Comment by slr71** [comment]
> Couples of things to look into:
> 
>     * Now is a good time to remove / replace `iplant`
> 
>     * Allow rich text formatting in message body

Sounds good. I probably won't tackle these things in this PR, but I'll add a Jira issue for myself so that I don't forget.

**Review by psarando — APPROVED** [review]
LGTM :shipit:

**Review by ianmcorvidae — APPROVED** [review]

**Comment by slr71** [comment]
Thanks for the reviews!

**Pull request merged**